src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingSubmissionIntegrationTest.java

```diff
@@ -179,33 +207,46 @@ void triggerBuildForExercise_Instructor() throws Exception {
 
         List<ProgrammingExerciseParticipation> participations = new ArrayList<>();
         for (ProgrammingSubmission submission : submissions) {
-            assertThat(submissionRepository.findWithEagerResultsById(submission.getId()).get().getLatestResult()).isNull();
+            var optionalSubmission = submissionRepository.findWithEagerResultsById(submission.getId());
+            assertThat(optionalSubmission.isPresent()).isTrue();
```

Comment:
```suggestion
            assertThat(optionalSubmission).isPresent();
```
AssertJ has special method for OptionalAssert, this an the simular places could be shortend like here.

Reply:
Ahh I see, thank you very much! I didn't know that :D I'll change it now